src/realm/object-store/schema.cpp

```diff
@@ -82,6 +82,18 @@ Schema::const_iterator Schema::find(ObjectSchema const& object) const noexcept
     return const_cast<Schema*>(this)->find(object);
 }
 
+Schema::iterator Schema::find(TableKey table_key) noexcept
+{
+    return std::find_if(begin(), end(), [&table_key](ObjectSchema os) {
```

Comment:
TableKey is just a thin wrapper around an int, so capturing it by reference is actually slower than capturing by value and just copying it.